config/roles.yml

```diff
@@ -255,3 +255,26 @@ point_in_time_full_access:
         - '*'
       allowed_actions:
         - 'manage_point_in_time'
+
+# Allows users to see security analytics detectors and others
+detectors_read_access:
```

Comment:
There already is a role called `alerting_read_access` | `alerting_full_access`, seem like they could be confusing for users trying to pick between the two, are there other names that might be more closely related to your plugin?

Reply:
the two changes start with the word "detectors". That has been the naming convention

Reply:
Should we call them ``security_analytics_read_access`` and so on?

Reply:
done